cli/src/test/java/com/liferay/blade/cli/command/ServerCommandsTest.java

```diff
@@ -177,6 +192,41 @@ public void run() {
 		Assert.assertFalse("Expected server start process to be destroyed.", pidProcess.isAlive());
 	}
 
+	private void _testServerStatus(boolean expectedServerStatus, String timeout) throws Exception {
```

Comment:
Can we move this into a separate Pull Request, addressing https://issues.liferay.com/browse/BLADE-338 (add "server status" command)?

Reply:
ok